packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -1621,27 +1622,28 @@ describe('postgresql partitioned tables introspection', () => {
-                                                                                                                                                                                                                        // 
-                                                                                                                                                                  `)
+                                                                                                                                                                                                                                // *** WARNING ***
+                                                                                                                                                                                                                                // 
+                                                                                                                                                                                                                                // The following models were ignored as they do not have a valid unique identifier or id. This is currently not supported by the Prisma Client.
+                                                                                                                                                                                                                                // - "measurement"
+                                                                                                                                                                                                                                // 
+                                                                                                                                                                                                                                // These tables are partition tables, which are not yet fully supported.
+                                                                                                                                                                                                                                // - Model "measurement"
+                                                                                                                                                                                                                                // 
+                                                                                                                                                                        `)
       expect(ctx.mocked['process.stdout.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
       expect(ctx.mocked['process.stderr.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
     })
   })
 
   describe('postgres partioned 27/2 - multiple partitioned tables found', () => {
+    const stopGap = 'partitioned'
```

Comment:
This `describe` and the previous one could be wrapped in another `describe` that share a `const StopGap` and `cost warningCode` maybe?

Reply:
I agree, let's leverage hierarchical `describe` blocks. Done